openedx/adg/conftest.py

```diff
@@ -124,3 +130,35 @@ def multilingual_course_group_with_courses(courses, course_group):
     MultilingualCourseFactory(course=courses['test_course2'], multilingual_course_group=course_group)
 
     return course_group
+
+
+@pytest.fixture(name='submitted_user_applications')
```

Comment:
Where are we using this fixture in this PR?

Reply:
Tests are going to be updated. Tests and a few other changes